cypress/e2e/CMS/$BlogSeries/index.js

```diff
@@ -0,0 +1,274 @@
+/// <reference types="Cypress" />
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+const siteSection = Cypress.env('test_site_section');
+
+function createRandomStr() {
+    var result = '';
+    var characters = 'abcdefghijklmnopqrstuvwxyz0123456789';
+    var charactersLength = characters.length;
+    for (var i = 0; i < 5; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+}
+let randomNum = createRandomStr();
+const frontEndBaseUrl = Cypress.env('front_end_base_url');
+
+And('user fills out the following fields', (dataTable) => {
+    for (const { fieldLabel, value, field_name } of dataTable.hashes()) {
+        if (fieldLabel === 'Pretty URL') {
+            value = `${value}-${randomNum}`;
+        }
+        cy.get(`input[name^='${field_name}']`).as('inputField').parent().find('label').should('include.text', fieldLabel);
+        cy.get('@inputField').type(value);
+    }
+});
+
+And('user clicks on title with url {string} from the list of content', (contentHref) => {
+    cy.get(`a[href='${siteSection}/${contentHref}-${randomNum}']`).click();
+});
+
+And('user selects a checkbox next to title with url {string} from the list of content', (url) => {
+    cy.get(`a[href='${siteSection}/${url}-${randomNum}']`).parent().parent().find('input.form-checkbox').check();
+});
+
+Given('user is navigating to the front end site with path site section plus {string}', (purl) => {
+    cy.visit(`${frontEndBaseUrl}${siteSection}/${purl}-${randomNum}`, { retryOnStatusCodeFailure: true });
+});
+
+And('the content item with url {string} does not exist in the list of content', (url) => {
+    cy.get(`a[href='${siteSection}/${url}-${randomNum}']`).should('not.exist');
+});
+
+And('user enters {string} as intro text', (introText) => {
+    cy.getIframeBody("iframe[title='Rich Text Editor, Intro Text field']").find('p').type(introText);
+});
+
+And('user fills out {string} text area with {string}', (textFieldLabel, value) => {
+    cy.getIframeBody("iframe[title^='Rich Text Editor, Body field']").find('p').type(value);
+});
+
+And('user uploads test file {string}', (fileName) => {
+    cy.fixture(fileName, { encoding: null }).as('fixture')
+    cy.get('input[type=file]').selectFile('@fixture');
+    cy.wait(4000);
```

Comment:
remove this wait and add this step in the feature file ( I commented there )

Reply:
done and removed the implementation of And user uploads test file  from index.js as it is in the common step already